packages/react-ui/components/Loader/Loader.tsx

```diff
@@ -102,9 +111,14 @@ export class Loader extends React.Component<LoaderProps, LoaderState> {
     if ((active && !prevProps.active) || prevProps.component !== component) {
       this.checkSpinnerPosition();
     }
+
+    if (prevProps.active !== active) {
+      this.toggleChildrenFocus();
```

Comment:
Давай избавимся от этого метода. Выглядит как оптимизация ради оптимизации =)
Логику лучше здесь явно написать, тем более такую короткую:
```suggestion
      if (active) {
        this.disableChildrenFocus();
      } else {
        this.enableChildrenFocus();
      }
```